docs/source/onnx.rst

```diff
@@ -73,9 +73,9 @@ exporter to print out a human-readable representation of the model::
     }
 
 You can also verify the output using the `ONNX <https://github.com/onnx/onnx/>`_ library,
-which you can install using `conda <https://anaconda.org>`_::
+which you can install using ``pip``::
 
-    conda install -c conda-forge onnx
+    pip install onnx
```

Comment:
Why do we need to change it to pip? Both are supported based on ONNX repository.

Reply:
I think pip is more standard way to install python packages. Since with conda onnx is provided in the conda-forge channel, it is not officially maintained and conda users should be able to find it just fine.